src/components/EntityInfo/EntityDefinitions.vue

```diff
@@ -0,0 +1,168 @@
+<template>
+  <dl id="entity-definitions">
+    <!-- Registration Date -->
+    <template v-if="!!registrationDate">
+      <dt class="mr-2">Registration Date:</dt>
+      <dd id="registration-date">{{ registrationDate }}</dd>
+    </template>
```

Comment:
Apologies for my cryptic comment earlier!  What do you think of making the following a component?

```javascript
  <template>
      <dt class="mr-2">{{ title }}:</dt>
      <dd :id="id">{{ value }}</dd>
    </template>
```

And this a component too?

```javascript
   <template>
      <dt class="mr-2">{{ title }}:</dt>
      <dd :id="id" class="cursor-pointer" @click="editBusinessProfile()">
        <span>{{ value }}</span>
        <v-btn :id="'change' + id + 'button'" small text color="primary">
          <v-icon small>mdi-pencil</v-icon>
          <span>Change</span>
        </v-btn>
      </dd>
    </template>
```

I know this may seem crazy to create tiny components like this but I think you'll see it makes our code more flexible.  By displaying these components by iterating over an array of data, it makes it very easy to change in the future if the business changes the requirements.

Reply:
We don't need that flexibility yet. Let's change it when/if we need to.

Right now I prefer to keep this entire "description list" together.